src/PuppeteerRunnerExtension.ts

```diff
@@ -357,7 +356,7 @@ async function getFrame(pageOrFrame: Page | Frame, step: Step): Promise<Frame> {
     'mainFrame' in pageOrFrame ? pageOrFrame.mainFrame() : pageOrFrame;
   if ('frame' in step && step.frame) {
     for (const index of step.frame) {
-      frame = frame.childFrames()[index];
+      frame = frame.childFrames()[index]!;
```

Comment:
I think we might actually have a bug here (not related to types) but if the frame does not exist we will probably fail on calling .childFrames()